docs/quickstart_pubFlow.md

```diff
@@ -1,12 +1,13 @@
 # Quickstart: Publisher flow
 
-## This is a sample on how a publisher is going to interact with the contracts.
-### Data Publisher is going to create a new NFT contract, then from there will deploy 3 new DTs used for different licensings, and then create a different type of go to market strategy for each DT. No Roles are used in this one, since the Data Publisher is the NFTOwner, he has all permissions.
+## This is a sample on how a data publisher is going to interact with the contracts.
 
+### DataPublisher is going to create a new NFT contract, then from there will deploy 3 new DTs used for different licensings, and then create a different type of go to market strategy for each DT. No Roles are used in this one, since the Data Publisher is the NFTOwner, he has all permissions.
 
 
 
-![alt text](https://github.com/oceanprotocol/contracts/blob/v4main/publisher.jpg?raw=true)
+
+![alt text](DataPublisherFlow.jpg)
```

Comment:
The image itself, and the text below, is formatted in a pretty ugly, un-maintainable fashion. Rather than fix that here, perhaps merge in as-is, and create a new issue for Akshay to clean this up?

Reply:
agreed